db/migrate/20230720211851_add_indices_to_display_ad_target_geolocations.rb

```diff
@@ -0,0 +1,15 @@
+class AddIndicesToDisplayAdTargetGeolocations < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def change
+    # Regular btree index
+    add_index :display_ads, :target_geolocations, algorithm: :concurrently
```

Comment:
Question, if this is stored as ltree, is regular index still necessary?

Reply:
I took a look into it and yeah actually the btree index accomplishes nothing since the column is an array (if the column was just a single ltree, my experience is that Postgres will use the btree index for some operations and the gist index for others)